viz/src/constants/sample-machines.ts

```diff
@@ -0,0 +1,22 @@
+import SampleMachines from "../../../our-machines/input-samples.json";
+
+const AllMachinesAsStrings = import.meta.globEager(
+  "../../../our-machines/*.json",
+  {
+    as: "raw",
+  }
+);
+
+export const sampleMachinesName = Object.keys(SampleMachines);
+
+export function getMachineCodeAndInput(machineName: string): {
+  machineCode: string;
+  input: string;
+} {
+  return {
+    machineCode: AllMachinesAsStrings[
```

Comment:
This is great ! I'm not sure we wanna purpose utm there.
As they usally take a long time to execute
They might just triggers halting guard